Bottom: 6ade27af770937b5cf95e0463911d2238046554e
Top:    261c4da8b3f597b154faea01549f67b52efb6d1e
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-10-20 21:33:22 +0100

Refresh of 3d-mouse-cursor-and-other-proj

---

diff --git a/src/hid/gtk/gui-output-events.c b/src/hid/gtk/gui-output-events.c
index c79673f..3805b78 100644
--- a/src/hid/gtk/gui-output-events.c
+++ b/src/hid/gtk/gui-output-events.c
@@ -1802,15 +1802,6 @@ ghid_port_drawing_area_expose_event_cb (GtkWidget * widget,
                 ghid_flip_y ? gport->view_y0 - PCB->MaxHeight :
                              -gport->view_y0, 0);
 
-#if 0
-  glBegin (GL_QUADS);
-  glVertex3i (region.X1, region.Y1, 0);
-  glVertex3i (region.X2, region.Y1, 0);
-  glVertex3i (region.X2, region.Y2, 0);
-  glVertex3i (region.X1, region.Y2, 0);
-  glEnd ();
-#endif
-
   // hid_expose_callback (&ghid_hid, global_view_2d ? &region : NULL, 0);
   ghid_draw_everything (&region);
